src/api.js

```diff
@@ -16,9 +16,7 @@ class ApiRoutes {
     this.callback = callback
     this.controllers = {}
     this.api = new Backend({
-      // Use the first server url as api root.
-      // @todo CDB2BGAZ-3755: Support multiple servers
-      apiRoot: root || OpenAPISpecification.servers[0].url,
+      apiRoot: root || '/',
       definition: OpenAPISpecification
```

Comment:
This is a breaking change, right?

Reply:
It can be breaking indeed, if you use the default value.
But if you use the `@ponbike/openapi-express` package, it set the root already.
I can change the version so it is clear that it can be break your app.

Reply:
I have change the version.
Also changed that you dont receive the `OpenAPIBackend` but just the `ApiRoutes` object that contains also the `OpenAPIBackend`. But now you can now receive also other information that is set, and i can write more/better tests.